sites/footers/old-footers-en.html

```diff
@@ -7,12 +7,86 @@
 	],
 	"altLangPage": "old-footers-fr.html",
 	"secondlevel": false,
-	"dateModified": "2025-04-30",
+	"dateModified": "2025-06-12",
 	"share": "true"
 }
 ---
 <div class="wb-prettify all-pre hide"></div>
 
+<h2>Version 8.1 (demoted)</h2>
+<ul>
+	<li>Added new link "Manage life events".</li>
+</ul>
+<p>Working example: <a href="demoted/footers-v8.1-en.html">Footer version 8.1</a></p>
+<details>
+	<summary>Expected output code</summary><pre><code>&lt;footer id="wb-info">
+		&lt;h2 class="wb-inv">About this site&lt;/h2>
```

Comment:
Fix indentation of the code sample. Everything is one tab too far, starting at line 23.